java/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/PartitionSender.java

```diff
@@ -61,7 +61,8 @@ public PartitionSender apply(Void a)
 	/**
 	 * Synchronous version of {@link #send(EventData)} Api. 
 	 * @param data the {@link EventData} to be sent.
-	 * @throws ServiceBusException if Service Bus service encountered problems during the operation.
+	 * @throws PayloadSizeExceededException    if the total size of the {@link EventData} exceeds a pre-defined limit set by the service. Default is 256k bytes.
```

Comment:
these Javadoc fixes are unrelated to this.. but included them as they are small & harmless.